docs/operator/notifications.md

```diff
@@ -0,0 +1,29 @@
+| When the replication rule transitioning into OK_STATE for each DATASET covered by the rule | DATASETLOCK_OK  | for dataset_lock in dataset_locks: payload = {'scope': dataset_lock.scope.external, 'name': dataset_lock.name, 'rse': get_rse_name(rse_id=dataset_lock.rse_id, session=session), 'rse_id': dataset_lock.rse_id, 'rule_id': rule.id} if dataset_lock.scope.vo != 'def': payload['vo'] = dataset_lock.scope.vo                                                                                                                                                                                                                                                                                                                           |
+| When the transfer is done                                                                  | transfer-done   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| When the transfer fail                                                                     | transfer-failed |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| When the deletion is done                                                                  | deletion-done   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
When a deletion is successful. There should be a payload as well.

Reply:
The sentence is pretty long (https://github.com/esilvaju/rucio/blob/6eac5dcc30cb6dac427cbc5070d46c170ae7fedf/lib/rucio/daemons/hermes/hermes.py#L390-L477), this is why I didn't added here.